rmw_fastrtps_cpp/src/subscription.cpp

```diff
@@ -242,6 +242,7 @@ create_subscription(
   info->dds_participant_ = dds_participant;
   info->subscriber_ = subscriber;
   info->topic_name_mangled_ = topic_name_mangled;
+  info->topic_ = topic_holder.topic;
```

Comment:
That is there so that we can call `get_inconsistent_topic_status` during a `SubListener::take_event`:
https://github.com/ros2/rmw_fastrtps/pull/654/files#diff-f393290aa0cfff761c6a3114fc26319b9edd9fd3c915db07165399b79e69394bR204